guides/v2.1/extension-dev-guide/versioning/dependencies.md

```diff
@@ -42,8 +43,11 @@ However, the dependency rules are different:
     ...
 }
 ```
```

Comment:
You could indent this block another 4 spaces so that it appears under the bolded text.